server.py

```diff
@@ -15,33 +17,65 @@ def setup():
     )
     sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
     sock.bind(addr)
-    sock.listen(1)
+    sock.listen(2)
     return sock
 
 
 def response_ok():
     """Return Header and Body information for Response 200."""
-    response_headers = 'HTTP/1.1 200 OK\nContent-Type: text/html'
-    response_body = '<html><body><h1></h1>'\
-        '<p>All good here, captain.</p></body></html>'
+    response_headers = (
+        b'HTTP/1.1 200 OK\r\n'
+        + date +
+        b'Content-Type: text/html\r\n'
+        b'Content-Length:\r\n')
```

Comment:
There's only one newline at the end of these headers, no blank line.  that's not properly formatted.  Also, there's no value set for `Content-Length`.